subprojects/gst-plugins-good/gst/rtp/gstrtph266pay.c

```diff
@@ -124,46 +124,51 @@ enum
   PROP_CONFIG_INTERVAL,
 };
 
-static void gst_rtp_h266_pay_finalize (GObject * object);
-static void gst_rtp_h266_pay_set_property (GObject * object, guint prop_id,
+// GObject methods
+static void _finalize (GObject * object);
+static void _set_property (GObject * object, guint prop_id,
     const GValue * value, GParamSpec * pspec);
-static void gst_rtp_h266_pay_get_property (GObject * object, guint prop_id,
+static void _get_property (GObject * object, guint prop_id,
```

Comment:
In my opinion, you should not follow https://fluendo.atlassian.net/wiki/x/A4BrJQ if working on an external project, in this case, GStreamer. Just follow GStreamer guidelines, and where there are no guidelines, just follow mimc the pattern of everything is there. If you look at code of `gstrtph26[4|5][de]pay` they use the `gst_rtp_h26[4|5]_[de]pay` prefix.

Reply:
> In my opinion, you should not follow https://fluendo.atlassian.net/wiki/x/A4BrJQ if working on an external project, in this case, GStreamer. Just follow GStreamer guidelines, and where there are no guidelines, just follow mimc the pattern of everything is there.

Agree :slightly_smiling_face: 

> If you look at code of gstrtph26[4|5][de]pay they use the gst_rtp_h26[4|5]_[de]pay prefix.

Indeed. Although, notice that while `h265(de)pay` is a C&P of `h264(de)pay`, this `h266pay` is completely different. I think it's different enough to justify a change in how methods are prefixed.

If it's OK with you, I'd like to give this a try. I'm curious to know if the community would accept this coding style.

Reply:
It's a very ugly coding style, IMO :p

Reply:
:broken_heart: 
The community is using it a lot :confused: These kinds of prefixes are useless IMO.